packages/@react-native-windows/cli/src/e2etest/projectConfig.utils.ts

```diff
@@ -12,37 +12,37 @@ export async function tryMkdir(dir: string): Promise<void> {
   } catch (err) {}
 }
 
-export async function ensureWinUI3Project(folder: string) {
+export async function ensureCppAppProject(folder: string, name: string, useWinUI3?: boolean, useHermes?: boolean, useExperimentaNuget?: boolean) {
   const windowsDir = path.join(folder, 'windows');
   if (fs.existsSync(windowsDir)) {
     await fs.rmdir(windowsDir, {recursive: true});
   }
   await tryMkdir(windowsDir);
 
   const replacements = {
-    name: 'WithWinUI3',
-    namespace: 'WithWinUI3',
+    name,
+    namespace: name,
     useMustache: true,
     projectGuidUpper: testProjectGuid,
     projectGuidLower: testProjectGuid.toLowerCase(),
-    useWinUI3: false,
-    useHermes: false,
-    useExperimentalNuget: false,
+    useWinUI3: !!useWinUI3,
```

Comment:
Out of curiosity what is the `!!` syntax for?

Reply:
It converts something "truthy" or "falsey" to a boolean. E.g."null" or "false" would both become "false". 

The same trick is common in C++ when interoperating with "C" APIs that use "1" and "0" for true and false.

Reply:
The first negation coerces the value to a boolean (that's the opposite of the value you want), and the second inverts it again.